src/plugins/saved_objects_management/server/routes/scroll_count.ts

```diff
@@ -39,34 +39,71 @@ export const registerScrollForCountRoute = (router: IRouter) => {
       validate: {
         body: schema.object({
           typesToInclude: schema.arrayOf(schema.string()),
+          namespacesToInclude: schema.maybe(schema.arrayOf(schema.string())),
           searchString: schema.maybe(schema.string()),
         }),
       },
     },
     router.handleLegacyErrors(async (context, req, res) => {
       const { client } = context.core.savedObjects;
+      const namespaces = [];
+      if (req.body.namespacesToInclude && req.body.namespacesToInclude.length > 0) {
+        req.body.namespacesToInclude.forEach((ns) => {
+          if (ns === null) {
+            namespaces.push('default');
```

Comment:
minor question, not sure it applies here - if there are multiple null ns, the array will contain multiple 'default', would that cause any issue?

Reply:
There is logic in the `namespace_service` to prevent this from happening: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/saved_objects_management/public/services/namespace_service.ts#L37-L39

I tried `findAll` with duplicates and it does work without issues, but because of the service registry process it won't happen.